panacea/web/static/js/app.js

```diff
@@ -91,28 +90,28 @@ const hideResults = () => {
   unidentifiedDrugsPanel.classList.add('hidden');
   ddisPanel.classList.add('hidden');
   errorPanel.classList.add('hidden');
-}
+};
 
 const displayDrugs = drugs => {
   const drugsTextElement = document.getElementById('drugs-text');
   drugsTextElement.innerHTML = JSON.stringify(drugs);
 
   drugsPanel.classList.remove('hidden');
-}
+};
 
 const displayUnidentifiedDrugs = drugs => {
   const unidentifiedDrugsTextElement = document.getElementById('unidentified-drugs-text');
   unidentifiedDrugsTextElement.innerHTML = JSON.stringify(drugs);
 
   unidentifiedDrugsPanel.classList.remove('hidden');
-}
+};
 
 const displayDdis = ddis => {
   const ddisTextElement = document.getElementById('ddis-text');
   ddisTextElement.innerHTML = JSON.stringify(ddis);
 
   ddisPanel.classList.remove('hidden');
-}
+};
 
 const displayError = data => {
```

Comment:
the destructuring is really nice - completely forgot it was a thing. thoughts on changing `data` to `{error}`?